.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:3030
```

Comment:
The thing is that at this stage I can't launch both api and project from the same port. Because for json server it is necessary to start the server on a different port. So I use a variable with a different port.